lib/ui/blocs/authentication/authentication_bloc.dart

```diff
@@ -67,18 +70,15 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
     try {
       final userProfileData = await _appSharedPrefs.getUserProfileData();
       final authData = await _secureStorageService.getUserAuthData();
+

```

Comment:
I guess we always assumed profile data available here...

Reply:
Yes, profile data is always there if the user is authenticated. 
Profile data only required field is account.